crates/nu-engine/src/external.rs

```diff
@@ -0,0 +1,36 @@
+/// Returns true if `name` refers to an external command
+
+#[cfg(target_arch = "wasm32")]
+pub fn is_external_cmd(#[allow(unused)] name: &str) -> bool {
+    true
+}
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Can we possibly not move this part to engine? I'm thinking of engine as, over time, containing things that should be more universal about Nushell.

This feels like something we should pass into the engine so that it has some callback that it can use to check if a command is a valid external or not.

Reply:
I have added a crate `nu-platform-specifics` containing this function.